parserscripts/phage.py

```diff
@@ -1,36 +1,36 @@
 import re
 
-class Phage:
-  supported_databases = {
-    # European Nucleotide Archive phage database
-    "ENA": r"^gi\|[0-9]+\|ref\|([^\|]+)\|\ ([^,]+)[^$]*$",
 
-    # National Center for Biotechnology Information phage database
-    "NCBI": r"^ENA\|([^\|]+)\|[^\ ]+\ ([^,]+)[^$]*$",
+class Phage:
+    supported_databases = {
```

Comment:
This is a constant, no? If so, should be upper case.

Reply:
> [Constants are usually defined on a module level and written in all capital letters with underscores separating words.](http://pep8.org/#constants)

It isn't defined at the module level and it is also mutable. I can change it if you want but I'm bit hesitant.

Reply:
One can define constants on a class-level as well if the encapsulation makes sense. Nothing stops a constant from being a mutable object; it's naming it with the conventional ALL_CAPS that says "don't modify me!".

Actually, since it's all convention, nothing stops you from modifying a constant that contains an immutable value either. 😉

Reply:
No problem, just wanted to check that it's what you guys wanted. Commit coming in now.